src/TLuaInterpreter.cpp

```diff
@@ -17645,3 +17647,87 @@ int TLuaInterpreter::announce(lua_State *L) {
     mudlet::self()->announce(text, processing);
     return 0;
 }
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#scrollTo
+int TLuaInterpreter::scrollTo(lua_State *L) {
+    QString windowName;
+    int targetLine;
+    bool stopScrolling = false;
+
+    int n = lua_gettop(L);
+    if (n > 2) {
+        lua_pushstring(L, "scrollTo: wrong number of arguments");
```

Comment:
I suggest just discarding the extra arguments and not error on it. Makes the API simpler to use.